server.js

```diff
@@ -91,15 +91,29 @@ app.patch('/api/v1/songs/:id', (request, response) => {
 });
 
 app.get('/api/v1/playlists', (request, response) => {
-  database('playlistSongs').select('playlist_id', 'playlist_name', 'songs')
-    .then((playlists) => {
-      response.status(200).json(playlists);
-    })
-    .catch((error) => {
-      response.status(500).json({ error });
-    });
+  let playlists = []
+  let songs = []
+  database('playlists').select(['playlists.id', 'playlists.playlist_name'])
+  .then((allPlaylists) => {
+    playlists = allPlaylists
+  })
+  database("songs")
+  .select(['songs.id', 'name', 'artist_name', 'genre', 'song_rating', 'playlist_songs.playlist_id'])
+  .join("playlist_songs", 'songs.id', 'playlist_songs.song_id')
+  .then((allSongs) => {
+    songs = allSongs;
+    for(let playlist of playlists) {
```

Comment:
@ameseee Does knex not have something to allow me to access the associated songs with a playlist?  Or we need to do this manually?  Please review, thanks!